### PR TITLE
New features in Remote Execution plugin 0.1.1-0.1.2

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
please do not merge until 0.1.2 is released (but feel free to review :-)

**Comment by ares** [comment]
0.1.2 was released Dec 17 (last year :-), could someone please read through?

File: plugins/foreman_remote_execution/0.1/index.md [hunk 1]
```diff
@@ -313,12 +341,60 @@ options
...
+template like this
+
+    Restart service %{service_name}
+
+If you set the input value to "httpd" the resulting description would become
+"Restart service httpd".
+
+You can also plan the execution in future or configure multiple rerunning.
+There's a second tab called **Scheduling** where you can choose from 
+* *Execute now* - immediate single execution,
```
Comment (domcleal) [inline plugins/foreman_remote_execution/0.1/index.md]: missing line break before this, preventing list formatting

File: plugins/foreman_remote_execution/0.1/index.md [hunk 2]
```diff
@@ -379,6 +458,73 @@ Alternatively, you may enable a fallback mode by setting
 consider any proxy associated with the host, such as its Puppetmaster, as long
 as the proxy also has a remote execution feature.
 
+# 4.2 Global settings
```
Comment (domcleal) [inline plugins/foreman_remote_execution/0.1/index.md]: `##` for correct heading level

File: plugins/foreman_remote_execution/0.1/index.md [hunk 3]
```diff
@@ -379,6 +458,73 @@ Alternatively, you may enable a fallback mode by setting
...
+  </tr>
+  <tr>
+      <td>remote_execution_ssh_user</td>
+      <td>Default user to use while the smart-proxy connects to the target using SSH. You may override per host by 
+      setting a parameter called remote_execution_ssh_user. Note that it can be set per Host, Host group, 
+      Operating system, Domain, Location and Organization. Also note that this can differ from effective user.</td>
+  </tr>
+</table>
+
+# 4.3 Permission delegation
```
Comment (domcleal) [inline plugins/foreman_remote_execution/0.1/index.md]: ditto, `##`

File: plugins/foreman_remote_execution/0.1/index.md [hunk 4]
```diff
@@ -379,6 +458,73 @@ Alternatively, you may enable a fallback mode by setting
...
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>remote_execution_effective_user</td>
+    <td>This is a login of default effective user for any job. When job is executed the effective user of the process
+    is changed accordingly (e.g. by sudo). This option can be overridden per job template and job invocation.</td>
+  </tr>
+  <tr>
+      <td>remote_execution_effective_user_method</td>
+      <td>What method should be used to set the effective user on target, currently only *su* and *sudo* are supported.</td>
```
Comment (domcleal) [inline plugins/foreman_remote_execution/0.1/index.md]: `<b>` or `<i>` inside HTML, markdown formatting doesn't work

**Comment by ares** [comment]
Thanks @domcleal for quick look, I've fixed all comments.

**Comment by domcleal** [comment]
Thanks @ares, merged.

**Pull request merged**